src/commands/plugins/generate.ts

```diff
@@ -1,6 +1,7 @@
-import Base from '../../command_base'
```

Comment:
Tslint rule: imports must be in alphabetical order and sorted by library, local path, etc.